src/LibraryManager.Vsix/UI/Controls/TargetLocation.xaml.cs

```diff
@@ -279,5 +286,26 @@ private void TargetLocation_LostFocus(object sender, RoutedEventArgs e)
                 Flyout.IsOpen = false;
             }
         }
+
+        private void LibraryNameChanged(object sender, PropertyChangedEventArgs e)
+        {
+            string targetLibrary = MutualPropertyChange.Instance.TargetLibrary;
+
+            this.Dispatcher.Invoke(() =>
+            {
+                if (TargetLocationSearchTextBox.Text.Equals(_lastTargetLocation))
```

Comment:
>Text [](start = 48, length = 4)

Can this ever be null?

Reply:
Bhavya thinks this should always be at least an empty string, not null.

---
In reply to: [200772612](https://github.com/aspnet/LibraryManager/pull/285#discussion_r200772612) [](ancestors = 200772612)